test-support/helpers/ember-tooltips.js

```diff
@@ -139,3 +139,14 @@ export function assertTooltipSide(assert, options = {}) {
       'Tooltip should be left of the target');
   }
 }
+
+export function assertContent(assert, contentString, options = {}) {
```

Comment:
To match existing test helper conventions, we should name this helper `assertTooltipContent`.